infrastructure/tasks/10-functions.ts

```diff
@@ -128,36 +136,36 @@ export const run = async () => {
   };
 
   const createdFunction = await webSiteClient.webApps.createOrUpdate(
-    (config as any).azurerm_resource_group,
-    (config as any).azurerm_functionapp,
+    config.azurerm_resource_group,
+    config.azurerm_functionapp,
     appConfig
   );
 
-  if ((config as any).azurerm_functionapp_slot && createdFunction.id) {
+  if (config.azurerm_functionapp_slot && createdFunction.id) {
     // Create slot for staging
     await webSiteClient.webApps.createOrUpdateSlot(
-      (config as any).azurerm_resource_group,
-      (config as any).azurerm_functionapp,
+      config.azurerm_resource_group,
+      config.azurerm_functionapp,
       appConfig,
-      (config as any).azurerm_functionapp_slot
+      config.azurerm_functionapp_slot
     );
 
     // Create git integration for the staging slot
-    if ((config as any).functionapp_git_repo) {
+    if (config.functionapp_git_repo) {
       await webSiteClient.webApps.createOrUpdateSourceControlSlot(
-        (config as any).azurerm_resource_group,
-        (config as any).azurerm_functionapp,
+        config.azurerm_resource_group,
+        config.azurerm_functionapp,
         {
-          branch: (config as any).functionapp_git_branch,
+          branch: config.functionapp_git_branch,
           deploymentRollbackEnabled: true,
-          // FIXME: `isManualIntegration: false` will fail trying to send an email
+          // FIXME: setting `isManualIntegration: false` will fail trying to send an email
```

Comment:
you should also open a story on pivotal for this

Reply:
https://www.pivotaltracker.com/story/show/152115927